src/software/backend/backend.cpp

```diff
@@ -0,0 +1,32 @@
+#include "software/backend/backend.h"
+
+void Backend::receiveTbotsRobotMsg(TbotsRobotMsg msg)
+{
+    SensorMsg sensor_msg;
+    *(sensor_msg.add_tbots_robot_msg()) = msg;
+    Subject<SensorMsg>::sendValueToObservers(sensor_msg);
```

Comment:
might as well save some duplication here and make this:
`receiveTbotsRobotMsgs({msg});`

Reply:
changed